manifests/a/Autodesk/DWGTrueView/24.2.72.0/Autodesk.DWGTrueView.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Autodesk.DWGTrueView
+PackageVersion: 24.2.72.0
+InstallerSwitches:
+  Silent: -suppresslaunch -d "C:\Temp\AutoDeskDWGTrueView"
+  SilentWithProgress: C:\Temp\AutoDeskDWGTrueView\Setup.exe -q
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://efulfillment.autodesk.com/NetSWDLD/2023/ACD/EC2FCD9E-AC4D-3E9D-8526-96832A231455/SFX/DWGTrueView_2023_English_64bit_dlm.sfx.exe?authparam=b91ba1b191a933c9c10b81c98913132b812b1b811b991b3389b98913a1c933b9&SESSION_ID=123456789;1083219296;2356937597;1;authparam;SESSION_ID

```

Comment:
As mentioned in the previous PR's

> The URL contains an authparam and a sessionID, which will likely expire and block future installations

Url's for winget should not contain authorization parameters, unless the publisher of the application has specifically authorized it. Please find a URL which is a Public URL and does not require an authorization parameter to download